static/app/views/replays/list/replayOnboardingPanel.tsx

```diff
@@ -276,7 +276,11 @@ export function SetupReplaysCTA({
       <ButtonList gap={1}>
         {renderCTA()}
         <Button
-          href="https://docs.sentry.io/product/session-replay/getting-started/"
+          href={
+            allMobileProj
+              ? 'https://docs.sentry.io/product/explore/session-replay/mobile/'
+              : 'https://docs.sentry.io/product/session-replay/getting-started/'
+          }
```

Comment:
```suggestion
          href={
            'https://docs.sentry.io/product/explore/session-replay/'
          }
```

Reply:
Or we could have diff links for each, if so let's use the updated docs link:
```suggestion
          href={
            allMobileProj
              ? 'https://docs.sentry.io/product/explore/session-replay/mobile/'
              : 'https://docs.sentry.io/product/explore/session-replay/web/'
          }
```

Reply:
@jas-kas specified in this ticket (https://github.com/getsentry/sentry/issues/74574) that we should be linking to the mobile specific docs. also for the other case if it's a combo of mobile + web projects then i don't want to force the user to the web docs right away, they should have the option of choosing

Reply:
makes sense! Since the current not allMobileProj link reroutes to https://docs.sentry.io/product/explore/session-replay/, could we update the link to that?